### DISCUSSION TITLE
ZeroMQ-based srsUe and srsGnb Multiple CPU Cores

### DISCUSSION THREAD

**Discussion by infinitydon** [srsran/srsRAN_Project#319] [General Help] [answered]
Hi, I will like to know if the ZMQ mode supports multiple CPU cores? If yes, how can I configure both the gNB and srsUE

**Comment by brendan-mcauliffe** [comment, marked as answer]
Hi @infinitydon, you can configure the number of threads used for certain processes for the UE and gNB in their associated configuration files, thus impacting the number of cores used. It doesn't matter if you are using ZMQ or UHD, you can configure these options. You can check the documentation for srsUE [here](https://docs.srsran.com/projects/4g/en/latest/usermanuals/source/srsue/source/index.html), and the gNB [here](https://docs.srsran.com/projects/project/en/latest/).

**Reply by infinitydon** [reply]
Thanks
